config/settings.yml

```diff
@@ -174,6 +174,12 @@ form_10_10cg:
       client_secret: ~
       timeout: 30
```

Comment:
This timeout value is not currently being used, since we pass in the async_timeout value explicitly.